### ISSUE TITLE
Convert csv to hdf5 issue: 'DataFrameArrays' object has no attribute 'dtype'

### ISSUE THREAD

**Issue by SophieLD** [vaexio/vaex#1252]
The problem is mentioned in following comment

**Comment by SophieLD** [comment]
HI, I have a csv file with more than 13million rows, I want to convert to hdf5:
I can run code:
df_chunk = vx.from_csv(r'df.csv', nrows=50_000_000)
but if I run following code:
df_chunk.export(r'df.hdf5')
I got error: AttributeError: 'DataFrameArrays' object has no attribute 'dtype'

Can you tell me what's wrong or how I can solve this. Thanks

**Comment by maartenbreddels** [comment]
Hi Sophie,

this should work, which version of vaex are you using?

Regards,

Maarten

**Comment by SophieLD** [comment]
Hi Maarten,

Thank you for your response. The problem was solved actually after I found out the version I used was so low (1.0.0b7) last night.

But I would like to share with you what I found (maybe only my computer has this issue). This was my first time using vaex, I followed the install code 'run conda install --name <conda_env_name> -c conda-forge vaex'  and only got installed version 1.0.0b7 into my environment, even I tried to upgrade, but did not work. So in order to install vaex 4.0, I created another environment with python 3.7 (my others env with python 3.8.3 or 3.8.5), then got vaex 4.0 installed and code worked in this new env.

Since these days I am working on very large size datasets, I probably will come up more questions by using vaex. :)
